source/upgrade.txt

```diff
@@ -220,6 +220,9 @@ The v3.11 driver introduces the following breaking changes to the build system:
 
 - Drops support for {+mdb-server+} v3.6.
 - Requires MongoDB C Driver v1.28.0 or later.
+- If the ``ENABLE_TESTS`` configuration option is set to ``ON``, you must configure
+  the driver to use C++14 or later. You can opt out of the C++14 requirement
```

Comment:
```suggestion
- If the ``ENABLE_TESTS`` configuration option is set to ``ON``, C++14 or later is required. You can opt out of the C++14 requirement
```

Suggest removing "you must configure". Some compilers enable C++14 by default and do not require explicit configuration. [Example](https://clang.llvm.org/cxx_status.html#:~:text=By%20default%2C%20Clang%2016%20or,in%20Clang%204%20and%20earlier):

>  By default, Clang 16 or later builds C++ code according to the C++17 standard.